autoAdjustHwloc.hpp

```diff
@@ -54,13 +54,13 @@ class autoAdjust
 
 		/* Destroy topology object. */
 		hwloc_topology_destroy(topology);
-		
+
 		printer::inst()->print_str("],\n\n**************** Copy&Paste END ****************\n");
 	}
 
 private:
 
-	inline void getConfig(hwloc_topology_t topology, hwloc_obj_t obj, size_t& numHashes, int& numCachesLeft)
```

Comment:
We need to keep the references, the changed values during moving over the tree of components must be passed to different hierarchies.